src/modal/modal-config.ts

```diff
@@ -62,6 +62,12 @@ export interface NgbModalOptions {
    * @since 1.1.0
    */
   backdropClass?: string;
+
+  /**
+   * Assign focus to the first focusable element when opened (true by default)
+   * @since 3.3.0
+   */
+  focusFirst?: boolean;
```

Comment:
My understanding was that we don't want to introduce this option for now. Did anything change?

Reply:
Did you see any new code push to this PR branch between my comment where I said I will remove it and now ? 😜

It will be removed...